source/NanostackRfPhyAT86RF215.cpp

```diff
@@ -1053,6 +1074,77 @@ static int rf_set_fsk_symbol_rate_configuration(uint32_t symbol_rate, rf_modules
     return 0;
 }
 
+static int rf_configure_by_ofdm_bandwidth_option(uint8_t option, uint32_t data_rate, rf_modules_e module)
+{
+    if (!option || option > 4) {
+        return -1;
+    }
+    uint32_t datarate_tmp = 100000;
+    for (int i=0; i<option-1; i++) {
+        datarate_tmp >>= 1;
```

Comment:
do we need for loop to this. I think 1 line should be possible.

Reply:
You are right. I'll change this.